### DISCUSSION TITLE
Way to restrict save location on drawio-desktop

### DISCUSSION THREAD

**Discussion by jtosuk** [jgraph/drawio-desktop#1257] [Q&A]
Hi,
There are quite a number of users within the organisation who would like to use drawio-desktop.
However due to company data policies, we don't want to let users save to certain locations such as google drive or dropbox.
Is there a way to restrict this with drawio-desktop please?
Either through the registry or config file(s) in the install directory on windows machines?
I am asking as many users work offsite without VPN and therefore it's not possible to restrict access using a firewall. It would be great if there's a way to disable the button/options saving to selected locations.
Many thanks.
JT

**Comment by davidjgraph** [comment]
drawio desktop doesn't have any options to save anywhere except for the local machine.
